.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
   - id: ruff
 
 - repo: https://github.com/codespell-project/codespell
-  rev: v2.2.5
+  rev: b8ecc9b3acf31690c3cb2fc5bb03a3fbbbc2d7a3
```

Comment:
Version supporting inline ignores has been merged but not released yet.

Reply:
Do you super badly need this? Let's not "pollute" this PR with such details, wdyt?

Reply:
@vblagoje if we don't bump that, we can't use inline ignores such as the one in line 262 of haystack/nodes/prompt/invocation_layer/amazon_bedrock.py. So we would need to globally ignore "tral" for spell corrections which is worse than just bumping codespell from my view.